pom.xml

```diff
@@ -91,6 +91,12 @@
             <artifactId>cucumber-core</artifactId>
             <version>${cucumber.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>3.0.7</version>
+            <scope>test</scope>
```

Comment:
remove <scope> everywhere in your pom

Reply:
but this dependency is used for web-services testing